numpy/ma/tests/test_core.py

```diff
@@ -997,8 +997,8 @@ def test_mvoid_multidim_print(self):
         assert_(str(t_2d[0]) == "([[1, --], [--, 4]], 1.0)")
         assert_(repr(t_2d[0]) == "([[1, --], [--, 4]], 1.0)")
 
-        t_ne = masked_array(data=[(1, (1, 1))],
-                            mask=[(True, (True, False))],
+        t_ne = masked_array(data = [(1, (1, 1))],
+                            mask = [(True, (True, False))],
```

Comment:
🙃 

```suggestion
        t_ne = masked_array(data=[(1, (1, 1))],
                            mask=[(True, (True, False))],
```

Reply:
All other keyword arguments in this function use ` = ` instead of `=`. I thought consistency might be more important than fixing everything, but if we are to apply this ruff rule some day, then we need to fix all occurrences...

Fixed now.